docs/core/linux-prerequisites.md

```diff
@@ -0,0 +1,343 @@
+        ```
+    * For Red Hat Enterprise 7 Workstation, use:
+         ```bash
+        subscription-manager repos --enable=rhel-7-workstation-dotnet-rpms
+        ```
+    * For Red Hat Enterprise 7 HPC Compute Node, use:
+         ```bash
+        subscription-manager repos --enable=rhel-7-hpc-node-dotnet-rpms
+        ```
+
+2. Install the scl tool.
+    ```bash
+    yum install scl-utils
+    ```
+3. Install .NET Core 1.1 (and all dependencies):
+    ```bash
+    yum install rh-dotnetcore11
+    scl enable rh-dotnetcore11 bash
+    ```
+4. Run the `dotnet --help` command to prove the installation succeeded.
+
+     ```bash
+     dotnet --help
+     ```
+> [!NOTE]
+> For Red Hat .NET channel access registration help, see [Chapter 1 of the .NET Core 1.1 Getting Started Guide](https://access.redhat.com/documentation/en/net-core/1.1/paged/getting-started-guide/) at Red Hat.
+
+
+## Install .NET Core for Ubuntu 14.04, 16.04, 16.10 & Linux Mint 17, 18 (64 bit)
+
+> [!Warning]
+> Before you start, remove any previous versions of .NET Core from your system.
+
+### Add the dotnet apt-get feed
+
+1. Set up the desired version host package feed.
+
+   **Ubuntu 14.04 / Linux Mint 17**
+    ```bash
+    sudo sh -c 'echo "deb [arch=amd64] https://apt-mo.trafficmanager.net/repos/dotnet-release/ trusty main" > /etc/apt/sources.list.d/dotnetdev.list' 
+    sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys B02C46DF417A0893
+    sudo apt-get update
+    ```
+    **Ubuntu 16.04 / Linux Mint 18**
+    ```bash
+    sudo sh -c 'echo "deb [arch=amd64] https://apt-mo.trafficmanager.net/repos/dotnet-release/ xenial main" > /etc/apt/sources.list.d/dotnetdev.list' 
+    sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys B02C46DF417A0893
+    sudo apt-get update
+    ```
+    **Ubuntu 16.10**
+    ```bash
+    sudo sh -c 'echo "deb [arch=amd64] https://apt-mo.trafficmanager.net/repos/dotnet-release/ yakkety main" > /etc/apt/sources.list.d/dotnetdev.list' 
+    sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys B02C46DF417A0893
+    sudo apt-get update
+    ```
+2. Install .NET Core.
+# [.NET Core 2.x](#tab/netcore2x)
+
+After host package feed setup, install .NET Core 2.0 on Ubuntu or Linux Mint:
+```bash
+sudo apt-get install dotnet-sdk-2.0.0
+```
+# [.NET Core 1.x](#tab/netcore1x)
+
+After host package feed setup, install .NET Core 1.x on Ubuntu or Linux Mint:
+```bash
+sudo apt-get install dotnet-dev-1.0.4
+```
+---
+3. Run the `dotnet --help` command to prove the installation succeeded.
+
+ ```bash
+     dotnet --help
+ ```
+## Install .NET Core for Debian 8 or 9 (64 bit).
+
+> [!Warning]
+> Before you start, remove any previous versions of .NET Core from your system.
+
+To install .NET Core on Debian 8 or 9 (64 bit):
+1. Get the prerequisites. 
+    ```bash
+    sudo apt-get install curl libunwind8 gettext
+    ```
+2. Download the .NET Core SDK binaries (tarball).
+
+# [.NET Core 2.x](#tab/netcore2x)   
```

Comment:
haven't tried but can this be indented? seems out of place with the surrounding code examples indented.